src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
```

Comment:
Since `background-repeat` property is already declared in `.stars__star` with the same value, it's redundant to repeat it here. It's a code style issue where we aim to avoid unnecessary repetition of code.